### PR TITLE
Updated package.json for test to remove vulnerable packages

### PR DESCRIPTION
**Pull request by hiyadav**

### CODE REVIEW

File: node/express/containerCosmosDBWithTests/Tests/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   },
   "dependencies": {
     "applicationinsights": "^1.0.0",
-    "gulp": "^3.9.1",
+    "gulp": "^4.0.2",
```
Comment (issacnitin) [inline node/express/containerCosmosDBWithTests/Tests/package.json]: This is breaking change, did you test if app is deployed?

Comment (dikhakha) [inline node/express/containerCosmosDBWithTests/Tests/package.json]: Do we need to increase the patch version  of the package

Comment (hiyadav) [reply node/express/containerCosmosDBWithTests/Tests/package.json]: That will not be required as we do not need to publish this separately somewhere and have always maintained version 0 for it.

Comment (hiyadav) [reply node/express/containerCosmosDBWithTests/Tests/package.json]: fixed that

Comment (hiyadav) [reply node/express/containerCosmosDBWithTests/Tests/package.json]: verified it using this DevOps project: https://portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/c00d16c7-6c1f-4c03-9be1-6934a4c49682/resourcegroups/VstsRG-hiyada-7247/providers/Microsoft.DevOps/pipelines/nodeexpresswithcosmos

**Pull request merged**